Neos.Neos/Classes/Domain/Service/FusionService.php

```diff
@@ -109,156 +71,35 @@ public function createFusionConfigurationFromSite(Site $site): FusionConfigurati
         return $this->fusionConfigurationCache->cacheFusionConfigurationBySite($site, function () use ($site) {
             $siteResourcesPackageKey = $site->getSiteResourcesPackageKey();
 
-            $siteRootFusionPathAndFilename = sprintf($this->siteRootFusionPattern, $siteResourcesPackageKey);
+            $siteRootFusionPathAndFilename = sprintf(self::SITE_ROOT_FUSION_PATTERN, $siteResourcesPackageKey);
 
             return $this->fusionParser->parseFromSource(
                 $this->fusionSourceCodeFactory->createFromNodeTypeDefinitions($site->getConfiguration()->contentRepositoryId)
                     ->union(
                         $this->fusionSourceCodeFactory->createFromAutoIncludes()
                     )
-                    ->union(
-                        $this->createSourceCodeFromLegacyFusionIncludes($this->prependFusionIncludes, $siteRootFusionPathAndFilename)
-                    )
                     ->union(
                         FusionSourceCodeCollection::tryFromFilePath($siteRootFusionPathAndFilename)
                     )
-                    ->union(
-                        $this->createSourceCodeFromLegacyFusionIncludes($this->appendFusionIncludes, $siteRootFusionPathAndFilename)
-                    )
             );
         });
     }
 
     /**
-     * Returns a merged Fusion object tree in the context of the given nodes
-     *
-     * @deprecated with Neos 8.3, will be removed with 9.0 {@link createFusionConfigurationFromSite}
-     *
-     * @param Node $startNode Node marking the starting point (i.e. the "Site" node)
-     * @return array<mixed> The merged object tree as of the given node
-     */
-    public function getMergedFusionObjectTree(Node $startNode)
-    {
-        return $this->createFusionConfigurationFromSite($this->findSiteBySiteNode($startNode))->toArray();
-    }
-
-    /**
-     * Create a runtime for the given site node
-     *
-     * @deprecated with Neos 8.3, will be removed with 9.0 use {@link createFusionConfigurationFromSite} and {@link RuntimeFactory::createFromConfiguration} instead
+     * Create a runtime for the given site
      *
      * @return Runtime
      */
     public function createRuntime(
-        Node $currentSiteNode,
-        ControllerContext $controllerContext
-    ) {
+        Site $site,
+        ActionRequest $actionRequest
+    ): Runtime {
```

Comment:
this method signature is not final and will further change with https://github.com/neos/neos-development-collection/issues/4086